packages/datagateway-dataview/cypress/integration/table/dls/visits.spec.ts

```diff
@@ -215,6 +215,23 @@ describe('DLS - Visits Table', () => {
       cy.contains('11.01 GB', { timeout: 10000 }).should('be.visible');
     });
 
+    it('and then calculate file size when the value is 0', () => {
+      // We need to wait for counts to finish, otherwise cypress
+      // might interact with the details panel too quickly and
+      // it re-renders during the test.
```

Comment:
Same here.

Reply:
No I think those are correct, its just the new intercept should be above the comment since I think it refers to ` cy.contains('[aria-rowindex="1"] [aria-colindex="4"]', '55').should(
        'exist'
      );`